TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/ParsedHttpMessagesAsDicts.java

```diff
@@ -139,13 +139,17 @@ private static Map<String, Object> convertRequest(@NonNull IReplayContexts.ITupl
         return makeSafeMap(context, () -> {
             var map = new LinkedHashMap<String, Object>();
             var message = HttpByteBufFormatter.parseHttpRequestFromBufs(byteToByteBufStream(data), true);
-            map.put("Request-URI", message.uri());
-            map.put("Method", message.method().toString());
-            map.put("HTTP-Version", message.protocolVersion().toString());
-            context.setMethod(message.method().toString());
-            context.setEndpoint(message.uri());
-            context.setHttpVersion(message.protocolVersion().toString());
-            return fillMap(map, message.headers(), message.content());
+            try {
```

Comment:
nit for the future (this is still an improvement as is): I'd like to surface something other than NullPointerException when message is null